tests/i18n/test_percents.py

```diff
@@ -4,19 +4,19 @@
 import os
 
 from django.template import Context, Template
-from django.test import SimpleTestCase, override_settings
+from django.test import override_settings
 from django.utils._os import upath
 from django.utils.encoding import force_text
 from django.utils.translation import activate, get_language, trans_real
 
-from .test_extraction import ExtractorTests
+from .test_extraction import SimplePoFileTests
 
 SAMPLEPROJECT_DIR = os.path.join(os.path.dirname(os.path.abspath(upath(__file__))), 'sampleproject')
 SAMPLEPROJECT_LOCALE = os.path.join(SAMPLEPROJECT_DIR, 'locale')
 
 
 @override_settings(LOCALE_PATHS=[SAMPLEPROJECT_LOCALE])
-class FrenchTestCase(SimpleTestCase):
+class FrenchTestCase(SimplePoFileTests):
```

Comment:
This doesn't seem correct as `SimplePoFileTests` no longer has any tests in it so now this subclass doesn't do anything.